src/torchcodec/_core/FFMPEGCommon.cpp

```diff
@@ -155,9 +156,77 @@ SwrContext* allocateSwrContext(
 #endif
 
   TORCH_CHECK(swrContext != nullptr, "Couldn't create swrContext");
+  status = swr_init(swrContext);
+  TORCH_CHECK(
+      status == AVSUCCESS,
+      "Couldn't initialize SwrContext: ",
+      getFFMPEGErrorStringFromErrorCode(status),
+      ". If the error says 'Invalid argument', it's likely that you are using "
+      "a buggy FFmpeg version. FFmpeg4 is known to fail here in some "
+      "valid scenarios. Try to upgrade FFmpeg?");
   return swrContext;
 }
 
+UniqueAVFrame convertAudioAVFrameSampleFormatAndSampleRate(
+    const UniqueSwrContext& swrContext,
+    const UniqueAVFrame& srcAVFrame,
```

Comment:
Nit: we should be consistent about `src` and `source`. I have a preference for `src`, as it's a universal abbreviation, particularly when paired with `dst`. But if we say `source` in a lot of other places, we should stick with that.

Reply:
Double nit, and I recognize this name already existed: `convertAudioAVFrameSampleFormatAndSampleRate()` is very long, and I feel like we're encoding parameter names that modify the operation into the name. I feel like it's clearer as just `convertAudioAVFrame()`.

Reply:
Sounds good, I'll merge as-is and follow-up with a PR to address these